htdocs/web_portal/views/downtime/add_downtime.php

```diff
@@ -494,14 +505,41 @@ function validateUtcDates(){
     }
 
     function getSitesServices(){
-        var siteId=$('#Select_Sites').val();
-        if(siteId != null){ //If the user clicks on the box but not a specific row there will be no input, so catch that here
-            $('#chooseEndpoints').empty(); //Remove any previous content from the endpoints select list
-            $('#chooseServices').load('index.php?Page_Type=Downtime_view_endpoint_tree&site_id='+siteId,function( response, status, xhr ) {
-                if ( status == "success" ) {
-                    validate();
-                }
-            });
+        let siteID = $('#Select_Sites').val();
+
+        /**
+         * If the user clicks on the box but not a specific row
+         * there will be no input, so catch that here.
+         */
+        if (siteID != null) {
+            // Remove any previous content from the endpoints select list.
+            $('#chooseEndpoints').empty();
+
+            let urlPath;
```

Comment:
I would suggest adding a comment here and also for the rest of the code below if needed as it's a little confusing what is happening and why.

Reply:
Ok. It shouldn't be an issue as I provided the name to be more descriptive.